### ISSUE TITLE
Inline Help

### ISSUE THREAD

**Issue by Elementari** [ysard/cookie-quick-manager#170]
Hello, 
I think it would be great if there were a button with a question mark for each command (delete ...) so the user can quickly find out what selecting a menu item will do. 
In particular (but not only) the context cookies command could also show advanced information on what's about to be deleted. The warning doesn really help if you don't know what's about to happen.

Thanks!

**Comment by ericjs** [comment]
I strongly agree this is needed. "Context cookies" is not even explained in the github readme.
